src/renderers/visual-editor.renderer.ts

```diff
@@ -15,6 +15,7 @@ export class VisualEditorRenderer extends DesktopRenderer {
   private async retrieveHtml(renderOpts: RenderOpts): Promise<any> {
     const { data, articleId, articleDetail, isMainPage } = renderOpts
 
+    /* istanbul ignore if */
```

Comment:
What's this for?

Reply:
Specify that next line is not covered by tests and this is expected.